content/300-guides/300-upgrade-guides/200-upgrading-versions/600-upgrading-to-prisma-5.mdx

```diff
@@ -0,0 +1,677 @@
+---
+title: 'Upgrade to Prisma 5'
+metaTitle: 'Upgrade to Prisma 5'
+metaDescription: 'Guides on how to upgrade to Prisma 5'
+tocDepth: 3
+toc: true
+---
+
+<TopBlock>
+
+Prisma 5.0.0 introduces a number of changes, including the usage of our new JSON Protocol, [which make Prisma faster by default](https://www.prisma.io/blog/prisma-and-serverless-73hbgKnZ6t#improved-startup-performance-by-9x). A full list of these changes can be found [in our release notes](https://github.com/prisma/prisma/releases/tag/5.0.0).
+
+This guide explains how upgrading might affect your application and gives instructions on how to handle breaking changes within Prisma 5.
+
+</TopBlock>
+
+## Upgrade the <inlinecode>prisma</inlinecode> and <inlinecode>@prisma/client</inlinecode> packages to Prisma 5
+
+To upgrade to Prisma 5 from an earlier version, you need to update both the `prisma` and `@prisma/client` packages.
+
+<Admonition type="alert">
+
+Before you upgrade, check each **breaking change** below to see how the upgrade might affect your application.
+
+</Admonition>
+
+<TabbedContent tabs={[<FileWithIcon text="npm" icon="file"/>, <FileWithIcon text="yarn" icon="file"/>, <FileWithIcon text="pnpm" icon="file"/>]}>
+
+<tab>
+
+```terminal
+npm update @prisma/client@latest
+npm update -D prisma@latest
+```
+
+</tab>
+
+<tab>
+
+```terminal
+yarn upgrade prisma@latest @prisma/client@latest
+```
+
+</tab>
+
+<tab>
+
+```terminal
+pnpm up prisma@latest @prisma/client@latest
+```
+
+</tab>
+
+</TabbedContent>
+
+## Version changes
+
+Prisma 5 includes some minimum version changes for Node.js, TypeScript, PostgreSQL, and SQLite. To use Prisma version 5.0.0 and up, you will need to have at least the minimum versions below:
+
+### Node.js minimum version change
+
+From Prisma version 5.0.0, the minimum version of Node.js supported is 16.13.0. If your project uses an earlier version of Node.js, you will need to upgrade it.
+
+See our [system requirements](/reference/system-requirements) for all minimum version requirements.
```

Comment:
This could theoretically be moved to the section above, before the "Node.js" headline. Here it is a bit out of place.